src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -289,6 +369,26 @@
                 event.preventDefault();
                 this.loadAction({projectId, versionId});
             },
+            handleEditVersionNote(projectId: number, versionId: string, versionNumber: number) {
+                this.projects.filter(project => {
+                    if (project.id === projectId) {
+                        this.displayProjectName = project.name
+                        this.editedNote = project.versions
+                            .find(version => version.versionNumber === versionNumber)!.note || ""
+                    }
+                })
```

Comment:
```suggestion
               const project = this.projects.find(p => p.id === projectId)!;
               this.displayProjectName = project.name;
               this.editedNote = project.versions
                            .find(version => version.versionNumber === versionNumber)!.note || ""
   
```
You could make this a little terser by using find to get the project (rather than filter) as well as to get the version. Something like this?